docker/Makefile

```diff
@@ -0,0 +1,18 @@
+TAG ?= $(shell git rev-parse --short HEAD)
+REPO_URL ?= $(shell cd ../terraform&&terraform output -json ecr-module | jq .ecr | jq -r .repository_url)
+#CONTAINER_NAME ?= wppj3
+
+.PHONY: login
+login:
+	@echo "Retrieving an authentication token and authenticate your Docker client to your registry"
+	aws ecr get-login-password --region ap-southeast-2 | docker login --username AWS --password-stdin ${REPO_URL}
```

Comment:
--username AWS? Can we have this as a parameter?

Reply:
This AWS CLI command provides authentication to local docker-cli.  You can these commands visiting your ECR repository on AWS Portal.